### PR TITLE
[NEW] Added endpoint to get the list of available oauth services

### LINKED ISSUE #8519: [NEW] API for retrieving all login services
It appears the only way we can get the login services is by subscribing to "meteor.loginServiceConfiguration", but that makes us receive the services one by one.

It would be better to have a method, preferably REST, to get all available login services instead of having to make that subscription.

### PR DESCRIPTION
**Pull request by MarcosSpessatto**
Closes #8519.
Added endpoint to get the list of available oauth services.

### CODE REVIEW

**Comment by MarcosSpessatto** [comment]
@rafaelks, could you take a look, please?

File: packages/rocketchat-api/server/v1/misc.js [hunk 1]
```diff
@@ -18,6 +18,26 @@ RocketChat.API.v1.addRoute('info', { authRequired: false }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('oAuthServices', { authRequired: false }, {
```
Comment (rafaelks) [inline packages/rocketchat-api/server/v1/misc.js]: What do you think about `settings.oauth`?

**Comment by MarcosSpessatto** [comment]
[Docs](https://github.com/RocketChat/docs/pull/638).

**Comment by rafaelks** [comment]
@MarcosSpessatto Thank you!!! 👊

**Comment by filipedelimabrito** [comment]
Cool! Thank you!

File: packages/rocketchat-api/server/v1/misc.js [hunk 2]
```diff
@@ -18,6 +18,26 @@ RocketChat.API.v1.addRoute('info', { authRequired: false }, {
...
+	get() {
+		const mountOAuthServices = () => {
+			const oAuthServicesEnabled = ServiceConfiguration.configurations.find({}).fetch();
+
+			return oAuthServicesEnabled.map((service) => {
+				return {
+					id: service._id,
+					name: service.service,
+					appId: service.appId || service.clientId
+				};
```
Comment (rodrigok) [inline packages/rocketchat-api/server/v1/misc.js]: Why not return `buttonLabelText, buttonLabelColor and buttonColor`?

**Pull request merged**